docs/specification/1.0.md

```diff
@@ -13,6 +13,14 @@ The CDS Hooks specification is available as an Open API Specification using Swag
 
 ## Discovery
 
+In order to allow EHRs to discover its services, a CDS service needs to provide a stable endpoint.  A CDS service MAY expose its services at 
```

Comment:
Per my below comment, I think this sentence is redundant once we move the original sentence up here.

What value do you see the additional example adding given that we have this same statement below as well? Additionally, it is `cds-services` (dash), not `cds_services` (underscore).